lib/warder/code_smell_runner.rb

```diff
@@ -1,9 +1,9 @@
 module Warder
-  # responsible for run code smells validation
-  class CodeSmellsRunner < Runner
+  # responsible for run code smell validation
+  class CodeSmellRunner < Runner
     CLI_OPTION = 's'
-    CLI_FULL_OPTION = 'code-smells'
-    DESCRIPTION = 'Run code smells validation'
+    CLI_FULL_OPTION = 'code-smell'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.